### PR TITLE
Abhishek user roles

### PR DESCRIPTION
**Pull request by Abhishek-chouksey**
@ssvaidyanathan  There is defect in a particular scenario. there was conflicts due to name changes in UserRoleMojo class. Thorough regression testing has now been done. Please do have a look .

Thank you.

### CODE REVIEW

File: src/main/java/com/apigee/edge/config/mavenplugin/UserRoleMojo.java [hunk 1]
```diff
@@ -218,7 +218,7 @@ public static String createUserRole(ServerProfile profile, String userRole) thro
 		UserRole userRoleObject = gson.fromJson(userRole, UserRole.class);
 		String namePayload = userRoleConversion(userRoleObject.name);
 
-		HttpResponse response = RestUtil.createOrgConfig(profile, userRole, namePayload);
+		HttpResponse response = RestUtil.createOrgConfig(profile, UserRoleMojo.userRole, namePayload);
```
Comment (ssvaidyanathan) [inline src/main/java/com/apigee/edge/config/mavenplugin/UserRoleMojo.java]: This looks good.. Please check line 250 and 280 in this file. Not sure what is the second argument being passed. It shuld be resource path in the Mgmt API. Please check other Mojo classes

There is reference to userRole but its being passed.

Comment (Abhishek-chouksey) [reply src/main/java/com/apigee/edge/config/mavenplugin/UserRoleMojo.java]: @ssvaidyanathan  Thanks for the review. 
The second argument being passed is resource path that is "userroles" string which we assign in class level variable userRole.

**Comment by Abhishek-chouksey** [comment]
@ssvaidyanathan  Thanks for approving changes, can you please mention when will my pull request get merge as we are waiting  to use it in production of our client.

Thanks once again for your review.

File: src/main/java/com/apigee/edge/config/mavenplugin/UserRoleMojo.java (hunk 1 above)
Comment (ssvaidyanathan) [reply src/main/java/com/apigee/edge/config/mavenplugin/UserRoleMojo.java]: Did you check line 250 and 280 as I mentioned ? Can you update it to `UserRoleMojo.userRole`

Comment (ssvaidyanathan) [reply src/main/java/com/apigee/edge/config/mavenplugin/UserRoleMojo.java]: Its ok - I will make the changes

**Comment by ssvaidyanathan** [comment]
Released 1.3.7
Please test and let me know

**Comment by Abhishek-chouksey** [comment]
@ssvaidyanathan Thanks for merging my pull request.
I tested, all things are working fine.
Thanks once again.

**Pull request merged**